documentation/src/pages/extensions/index.tsx

```diff
@@ -52,13 +54,21 @@ export default function HomePage() {
           </p>
         </div>
 
-        <div className="search-container">
-          <input
-            className="bg-bgApp font-light text-textProminent placeholder-textPlaceholder w-full px-3 py-3 text-[40px] leading-[52px] border-b border-borderSubtle focus:outline-none focus:ring-purple-500 focus:border-borderProminent caret-[#FF4F00] pl-0"
-            placeholder="Search for extensions"
-            value={searchQuery}
-            onChange={(e) => setSearchQuery(e.target.value)}
-          />
+        <div className="flex justify-between items-center mb-8">
+          <div className="search-container flex-1">
+            <input
+              className="bg-bgApp font-light text-textProminent placeholder-textPlaceholder w-full px-3 py-3 text-[40px] leading-[52px] border-b border-borderSubtle focus:outline-none focus:ring-purple-500 focus:border-borderProminent caret-[#FF4F00] pl-0"
+              placeholder="Search for extensions"
+              value={searchQuery}
+              onChange={(e) => setSearchQuery(e.target.value)}
+            />
+          </div>
+          <Link to="/extensions/recipe-generator" className="no-underline ml-4">
```

Comment:
Wondering if there's a better place to put the link to the Recipe generator. Instead of where it is under extensions. Perhaps a good time to explore adding a resources nav bar menu that lists the `install-link generator` and then the `recipe-generator`